### DISCUSSION TITLE
Copilot is not working (Thank you for participation...)

### DISCUSSION THREAD

**Discussion by S2LF** [community/community#44662] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hi !

I bought Copilot yesterday.

It worked fine but today I need to login again and I doesn't work.

I find same bug like me in this discussion : https://github.com/community/community/discussions/39540 and this one : https://github.com/orgs/community/discussions/40022#discussioncomment-4234435

It look like it was fixed but it's not.

In other part, I don't have any confirmation of my billing of Copilot.
In Settings/Copilot : it seems I can configure
![image](https://user-images.githubusercontent.com/49229770/213164999-bbd466f5-31fc-429d-bcf7-474f11ec4bdd.png)

In Settings/Billings and plan : There no "edit" button to change subscription
![image](https://user-images.githubusercontent.com/49229770/213165160-3b2ce266-1a39-4ec0-a89c-8a17b8dd3163.png)

In VSCode Output : 
![image](https://user-images.githubusercontent.com/49229770/213165782-c8a40b93-f3b6-4603-9a52-68cba1e6c66b.png)

Is there anyway to fix it ? Am i doing something wrong ?
The stranger thing is I used it yesterday but today it doesn't work 🤔 

Thanks

**Comment by Matteo-it** [comment]
Hi, write to GitHub Support to get help and fix your issue!

**Reply by S2LF** [reply]
Done,
there is the link to the ticket (if someone need to follow) : https://support.github.com/ticket/personal/0/1972664

Thanks

**Comment by riderx** [comment]
Same issue here, i did even paid and still not working 🤣
